File: src/main/java/com/kfgs/policypapers/service/policypapersService.java
package com.kfgs.policypapers.service;

import com.kfgs.domain.TbPolicyDocument;

import java.util.List;
import java.util.Map;

public interface policypapersService {

    public Map<String, List<TbPolicyDocument>> getList();

    public Map<String,Object> getList(Map searchMap);

    public int updateByExampleSelective(Map pData);

    Map<String,Object>  selectByPrimaryKey(Map pData);

}

File: src/main/webapp/js/service/policyPapersService.js
//统计service层
policyPapersapp.service('policyPapersService',function ($http) {

    this.search = function (searchMap) {
        return $http.post('policypapers/getList.do',searchMap);
    }

    this.initById = function (pData) {
        return $http.post('policypapers/getContent.do',pData);
    }
});